src/app/profile/profile.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-profile',
+  templateUrl: './profile.component.html',
+  styleUrls: ['./profile.component.scss']
+})
+export class ProfileComponent implements OnInit {
+  @Input() name;
+  @Input() desc;
+  @Input() dname;
```

Comment:
I would suggest using more descriptive names like `description` and `displayName`. Also, it is considered a good practice to mention the type of the variable like `@Input() name: string;`.